schemas/iso19139/src/main/plugin/iso19139/formatter/xsl-view/view.xsl

```diff
@@ -952,4 +988,55 @@
   <xsl:template mode="render-value"
                 match="@*"/>
 
-</xsl:stylesheet>
+  <!-- isBasedOn parentidentifier -->
+
+  <xsl:template mode="render-value"
+                match="gmd:parentidentifier/gco:CharacterString">
+    <span itemprop="isBasedOn">
+      <a href="{concat($nodeUrl, 'api/records/', .)}"><xsl:value-of select="."/></a>
+    </span>
+  </xsl:template>
+
+  <!-- license -->
+  <xsl:template mode="render-value"
+                match="gmd:MD_LegalConstraints/gmd:otherConstraints/gmx:Anchor/@xlink.href">
+    <span itemprop="license"><a href="{.}"><xsl:value-of select="."/></a></span>
+  </xsl:template>
+
+  <!-- spatialCoverage	Place 
+  temporalCoverage	DateTime -->
+  <xsl:template mode="render-value"
+                match="gmd:extent/gml:TimePeriod/gml:beginPosition">
```

Comment:
Maybe to remove to the main template 
```
 <xsl:template mode="render-value"
                match="gco:Integer|gco:Decimal|
       gco:Boolean|gco:Real|gco:Measure|gco:Length|gco:Distance|gco:Angle|gmx:FileName|
       gco:Scale|gco:Record|gco:RecordType|gmx:MimeFileType|gmd:URL|
       gco:LocalName|gml:beginPosition|gml:endPosition">
```
?

Reply:
same here, to add the itemprop based on context? i'm curious to see how that would look.+1